org.osgi.service.feature/src/org/osgi/service/feature/Features.java

```diff
@@ -13,54 +13,41 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * SPDX-License-Identifier: Apache-2.0 
+ * SPDX-License-Identifier: Apache-2.0
  *******************************************************************************/
 package org.osgi.service.feature;
 
+import org.osgi.annotation.versioning.ProviderType;
+
```

Comment:
Why is this moved? Are you using the Eclipse formatting configured for this project? We should not add these changes which formatting will change again. I am not saying you must use Eclipse to work on the code. But you still need to not change the code in ways that will change it again when formatted.

Reply:
Fixed

Reply:
You should squash the commits into one commit.